code/modules/events/wizard/magicarp.dm

```diff
@@ -36,11 +36,12 @@
 	projectilesound = 'sound/weapons/emitter.ogg'
 	maxHealth = 50
 	health = 50
+	gold_core_spawnable = NO_SPAWN
 	random_color = FALSE
 	var/allowed_projectile_types = list(/obj/item/projectile/magic/change, /obj/item/projectile/magic/animate, /obj/item/projectile/magic/resurrection,
-	/obj/item/projectile/magic/death, /obj/item/projectile/magic/teleport, /obj/item/projectile/magic/door, /obj/item/projectile/magic/aoe/fireball,
-	/obj/item/projectile/magic/spellblade, /obj/item/projectile/magic/arcane_barrage)
-	
+	/obj/item/projectile/magic/teleport, /obj/item/projectile/magic/door, /obj/item/projectile/magic/aoe/fireball, /obj/item/projectile/magic/spellblade,
+	/obj/item/projectile/magic/arcane_barrage, /obj/item/projectile/magic/death)
```

Comment:
revert? this shouldn't have changed and it'll show up on the tracker as a change.

Reply:
Yeah, makes sense considering it basically stayed the same there.